### DISCUSSION TITLE
Captcha only for logged out users?

### DISCUSSION THREAD

**Discussion by dietrichmd** [liedekef/events-made-easy#188] [Q&A] [answered]
Hey Franky...Is there a way to only show Captcha on RSVP forms for logged-out users?  Currently, using [EME_IF] to hide the captcha still fails on submit because the form requires it (as set in the backend.)

Is there a way to do what I'm trying to do in the current version?  Is this something that could be added as a hook or filter somewhere?  

Thanks.

**Comment by liedekef** [comment, marked as answer]
Currently this is not possible: if you require the captcha, it is for everybody. Your idea is interesting though :-)

**Reply by dietrichmd** [reply]
Here's to hoping I see this in a future release :)

**Reply by liedekef** [reply]
The dev-version should now have this.
